pkg/grpc/grpc.go

```diff
@@ -50,6 +51,11 @@ func NewGRPCClientFromConfiguration(configuration *configuration.GRPCClientConfi
 // network addresses of UNIX socket paths provided.
 func NewGRPCServersFromConfigurationAndServe(configurations []*configuration.GRPCServerConfiguration, registrationFunc func(*grpc.Server)) error {
 	serveErrors := make(chan error)
+
+	if configurations == nil {
```

Comment:
`len(configurations) == 0`

It is possible for a slice to have length zero, but not point to a nil memory address.

Reply:
Fair point, the change now works as expected with `nil` or `[]`